cmd/restic/cmd_key.go

```diff
@@ -212,6 +212,10 @@ func runKey(ctx context.Context, gopts GlobalOptions, args []string) error {
 
 	switch args[0] {
 	case "list":
+		if gopts.NoLock {
+			return listKeys(ctx, repo, gopts)
```

Comment:
I'd prefer to just wrap the `lockRepo` and `unlockRepo` call into an if block, in order to not duplicate the `listKeys` call.

Reply:
ok